Game of Life.py

```diff
@@ -0,0 +1,31 @@
+# TC : O(n)
```

Comment:
Kindly check the complexity.

Reply:
Here by n I meant number of elements in the board. Its supposed to be O(m*n)  in actual terms.